src/SEOManager.php

```diff
@@ -224,9 +224,13 @@ public function favicon(): static
     }
 
     /** Append canonical URL tags to the document head. */
-    public function withUrl(): static
+    public function withUrl(?string $origin = null): static
```

Comment:
Is origin the right term here, or would canonical be more accurate?

Reply:
I think either would be fine. But this RFC says the scheme/host/port combination is an origin.

https://www.rfc-editor.org/rfc/rfc6454#section-5